### PR TITLE
Add torchapp v0.3.6

### PR DESCRIPTION
**Pull request by vinisalazar**
This PR supersedes #23427.

Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by vinisalazar** [comment]
@conda-forge/help-python please review and merge.

File: recipes/torchapp/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/torchapp-{{ version }}.tar.gz
+  sha256: 8084f005dbd3548aa1ffb341b6539325adc475b5d9cfcb5967d4af45ab2e9ff2
+
+build:
+  entry_points:
+    - torchapp = torchapp.main:app
+    - torchapp-imageclassifier = torchapp.examples.image_classifier:ImageClassifier.main
+  skip: true  # [win]
```
Comment (ocefpaf) [inline recipes/torchapp/meta.yaml:16]: If the reason for skipping Windows is a missing dependency, we still prefer to make this noarch (if possible) and let Windows fail. That means this package would be ready when the missing dependency is available without the need for any changes here.

Comment (vinisalazar) [reply recipes/torchapp/meta.yaml:16]: @ocefpaf thank you for the review. This is the error I get with Windows:

```
conda_libmamba_solver.exceptions.ExplainedDependencyNeedsBuildingError: Unsatisfiable dependencies for platform win-64: {MatchSpec("pytorch[version='>=1.12.1']"), MatchSpec("torchvision[version='>=0.13.0']"), MatchSpec("fastai==2.7.10=pyhd8ed1ab_0"), MatchSpec(">=1.7,<1.14")}
Encountered problems while solving:
  - nothing provides requested pytorch >=1.12.1
  - nothing provides requested torchvision >=0.13.0
  - nothing provides pytorch >=1.7,<1.14 needed by fastai-2.7.10-pyhd8ed1ab_0
```

Could I revert the PR back to [`7dad632`](https://github.com/conda-forge/staged-recipes/pull/24165/commits/7dad632d28f0c05b2417d2a5fb3d3a15ba0f4ae2) then?

Comment (ocefpaf) [reply recipes/torchapp/meta.yaml:16]: 👍

Comment (vinisalazar) [reply recipes/torchapp/meta.yaml]: @ocefpaf done. Please merge when ready. Thanks!

**Comment by vinisalazar** [comment]
Ready to merge (Windows will fail but keep it noarch).

**Pull request merged**